sources/packages/backend/apps/api/src/route-controllers/student-appeal/models/student-appeal.dto.ts

```diff
@@ -1,20 +1,29 @@
 import { Type } from "class-transformer";
 import {
+  ArrayMaxSize,
   ArrayMinSize,
   IsDefined,
   IsEnum,
   IsNotEmpty,
   IsPositive,
+  MaxLength,
   ValidateNested,
 } from "class-validator";
 import { StudentAppealStatus } from "@sims/sims-db";
+import { JsonMaxSize } from "../../../utilities/class-validation";
+
+const JSON_10KB = 10240;
```

Comment:
The constants FORM_NAME_MAX_LENGTH and JSON_10KB can be global and not just for the scope of student appeal DTO.

Reply:
Where to put it?

Reply:
Sorry I mentioned that in the next comment. 
We can declare them in a constant file created in `route-controllers/utils`

Reply:
Usually, we create `constants` folders as needed with specific constant files.
We already have one under `api/constants`. We can add specific files there and just add it to the index.

Reply:
ok. Done. Thanks!